src/bin/registerRule.js

```diff
@@ -49,14 +53,14 @@ program
     () => {
       console.log('');
       console.log('Arguments:');
-      console.log('  config       path to a config file');
-      console.log('  ruleName     Rule name to be registered');
-      console.log('  ruleAddress  Rule address to be registered');
-      console.log('  ruleAbi      Abi of the rule');
+      console.log('  config           path to a config file');
+      console.log('  ruleName         rule name to be registered');
+      console.log('  ruleAddress      rule address to be registered');
+      console.log('  ruleAbiFilePath  file path containing abi of the rule');
       console.log('');
       console.log('Examples:');
       console.log('  Register rule in TokenRules:');
-      console.log('  $ registerRule.js config.json ruleName 0xa4aa50fbd4767085705db09e020a781e58e2fbf2 ruleAbi');
+      console.log('  $ registerRule.js config.json ruleName 0xa4aa50fbd4767085705db09e020a781e58e2fbf2 ruleAbiFilePath');
```

Comment:
Provide valid example for `ruleName` and `ruleAbiFilePath`.
Example :-
`ruleName` ==> `PricerRule`
`ruleAbiFilePath` ==> `./rules/PricerRule`

Reply:
Done!